data/json/recipes/food/frozen.json

```diff
@@ -7,7 +7,7 @@
     "skill_used": "cooking",
     "difficulty": 1,
     "time": "10 m",
-    "autolearn": true,
+    "book_learn": [ "dairy_book", 1 ],
```

Comment:
Why do you think people need a book recipe to figure out milkshakes?

Reply:
I'm going to honestly say this one was not super scientific.  I asked myself if I knew how to make a milkshake without ice-cream and couldn't think of the steps, then I asked a coworker if they knew how to make a milkshake without ice-cream and she said "yes" I asked her to describe the steps and she couldn't.  I felt like Milkshakes are easy in theory and more difficult in practice.  How much sugar should a milkshake have?  We don't really have a good system for recipes that a survivor could figure out by wasting a bunch of ingredients trying to recreate something they've never made for themselves pre-Cataclysm.

Reply:
Talked to a bunch of randys and decided to autolearn it back in.